low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -73,32 +264,23 @@ def name():
         return ServiceMonitor.SENSOR_NAME
 
     def __init__(self):
-        """Initialize the relavent datastructures."""
+        """Initialize the relevant datastructures."""
         super(ServiceMonitor, self).__init__(self.SENSOR_NAME,
-                                                self.PRIORITY)
-
-        self.services_to_monitor = copy.deepcopy(
-            Conf.get(SSPL_CONF, f"{self.SERVICEMONITOR}>{self.MONITORED_SERVICES}", [])
-        )
+                                             self.PRIORITY)
 
-        self.not_active_services = {
-            # 'service_name' : [timestamp, prev_state, prev_substate],
-            # timestamp => instance when transition to non_active state occurred.
-            # prev_state => state from which it came to non_active state
-            # prev_substate => substate from which it came to non_acitve substate.
-        }
-        self.failed_services = []
+        self.services_to_monitor = set(Conf.get(
+            SSPL_CONF, f"{self.SERVICEMONITOR}>{self.MONITORED_SERVICES}",
```

Comment:
On conf read failure, if default service list is given blank, why sensor needs to run even?

Reply:
Ok. shutting down the sensor if it is empty